e2e-tests/trade-finance-manager/cypress.json

```diff
@@ -16,7 +16,7 @@
   "testFiles": "**/*.spec.js",
   "responseTimeout": 100000,
   "pageLoadTimeout": 120000,
-  "redirectionLimit": 50,
+  "redirectionLimit": 60,
```

Comment:
do we need this?

Reply:
Yes, because an additional task has been added. One test for example checks for completing each task in sequential order. This means that each task goes into a new route (form) and the back to the tasks page. For a manual deal in one test spec it'll redirect at least 20 times for example. 

Without this change cypress throws an error saying there's too many redirects 👍